styles/components/_button.scss

```diff
@@ -76,8 +76,19 @@
     }
   }
 
+  &.-primary-link { 
+    @include btn-variant($brand-white, $brand-sidebar-lt);
+    border: 1px solid $brand-gray;
+
+    &:hover,
+    &:focus {
+      color: $brand-white;
+      background: $brand-sidebar-lt;
+    }
```

Comment:
I have no solid opinion about this. But OK. Anyway, was this instructed or did you come up with it? :ok_hand: